### DISCUSSION TITLE
Git LFS has been disabled

### DISCUSSION THREAD

**Discussion by kripper** [community/community#51525] [Projects and Issues]
### Select Topic Area

Question

### Body

Today we moved our OS project to an organization.
Minutes later, we received this mail:

```
Git LFS has been disabled on the organization RosettaDrone because you’ve exceeded your data plan by at least 150%. Please purchase additional data packs to cover your bandwidth and storage usage:

  https://github.com/organizations/RosettaDrone/billing/data/upgrade

Current usage as of 30 Mar 2023 07:36PM CEST:

  Bandwidth: 0.0 GB / 1 GB (0%)
  Storage: 2.5 GB / 1 GB (250%)
```

We won't pay, since it's a free OpenSource project.
We deleted the big files, but Storage is not being recovered.
Deleting and recreating the repo and loosing all project info, issues, etc. is not a solution.

Please purge the removed objects contained in `rosettadrone\app\src\main\cpp\lib`.

**Comment by tusharhqq** [comment]
It seems that your organization has exceeded the data plan for Git LFS (Large File Storage) on GitHub, and the storage usage is currently at 250% of the allotted space. Deleting the large files is a good first step, but it appears that the storage has not been recovered.

To address this issue, you can try purging the removed objects contained in the directory rosettadrone\app\src\main\cpp\lib. This will help to remove any unused or unnecessary files that may still be taking up space. Here are the steps to do this:

Navigate to the directory rosettadrone\app\src\main\cpp\lib.
Identify any files that have been deleted but are still present in the directory.
Use the Git LFS command git-lfs prune to remove any unused objects. This command will remove any objects that are not referenced in the Git LFS history.
After running the prune command, run the Git command git gc to perform a garbage collection and free up any additional space.
If these steps do not resolve the issue, you can also try contacting GitHub support to see if they can assist you with resolving the storage overage without requiring a paid upgrade.

**Comment by kripper** [comment]
Hi @tusharxoxoxo,

I tried to prune, but many files couldn't be pruned.
Then I rewrote the history with `git filter-branch` and tried `git push --force`, but it failed becuase...we have no storage left :-(

I also tried to contact github support.

**Comment by kripper** [comment]
If I do `git filter-branch` + `git push`, will all other developers have to clone again?
That's not something we want to go through.

Found this:
https://stackoverflow.com/questions/34579211/how-to-delete-a-file-tracked-by-git-lfs-and-release-the-storage-quota

And this comment:

> "Unbelievable. A single bad push of a too-large LFS file and now my repository is permanently locked unless I either pay them $5 a month in perpetuity or I delete and migrate the entire repository. What an incredibly consumer-unfriendly and exploitative business practice"

**Comment by kripper** [comment]
GitHub team purged our LFS.

**Reply by tusharhqq** [reply]
Nice
